ibr/png-parser/REDAME.md

```diff
@@ -0,0 +1,16 @@
+## Instructions for Running the PNG Parser:
+
+1. Compile the proto definition to python classes:
+	- Install latest release of protocol buffers
+	    - go to [https://developers.google.com/protocol-buffers/docs/downloads]()
+	    - click on latest version release page
+	    - click on protobuf-$VERSION-$PLATFORM.zip to download the protocol buffers pre-built binary
+	    - unzip downloaded file
+	    - place bin/protoc somewhere in your $PATH
```

Comment:
could we use a pip to manage this instead of manually downloading source?

Reply:
According to the protobuf official website, there are two parts of protobuf that need to be installed: Protocol Compiler and language specific Runtime library. Here we used pip to install the runtime library, but the simplest way to install the protocol compiler is to download a pre-built binary from the release page.

Reply:
sg, thanks for looking into this :)